docs/howtos/releases.md

```diff
@@ -0,0 +1,99 @@
+# Application Services Release Process
+
+## Nightly builds
+
+Nightly builds are automatically generated using a taskcluster cron task.
+
+- The results of the latest successful nightly build is listed here:
+  https://firefox-ci-tc.services.mozilla.com/tasks/index/project.application-services.v2.nightly/latest
+- The latest nightly decision task should be listed here:
+  https://firefox-ci-tc.services.mozilla.com/tasks/index/project.application-services.v2.branch.main.latest.taskgraph/decision-nightly
+- If you don't see a decision task from the day before, then contact releng.  It's likely that the cron decision task is broken.
+
+## Release builds
+
+Release builds are generated from the `release-vXXX` branches.  Whenever a commit is pushed there,
+we build all artifacts needed for the release.  Once we're ready to publish the release, we run the
+taskcluster release promotion action which signs and publishes the artifacts.
+
+TODO: explain release-promotion more.
+
+## What to do at the end of a nightly cycle
+
+1. Start this 2 workdays before the nightly cycle ends
+
+2. Run the `automation/prepare-release.py` script.  This should:
+
+ * Create a new branch named `release-vXXX`
+ * Create a PR against that branch that updates `.buildconfig-android.yml` like this:
+
+```
+diff --git a/.buildconfig-android.yml b/.buildconfig-android.yml
+index 8cd923873..6482018e0 100644
+--- a/.buildconfig-android.yml
++++ b/.buildconfig-android.yml
+@@ -1,4 +1,4 @@
+-libraryVersion: 114.0a1
++libraryVersion: 114.0
+ groupId: org.mozilla.appservices
+```
+
+ * Create a PR on `main` that starts a new CHANGELOG haeder.
+
+3. Trigger release-promotion once the PRs are approved, merged, and CI has completed
+
+ * TODO: explain how to do this
```

Comment:
Ideally this will be done via shipit, but until then we may want only Releng to have permission to do this. This way it acts as a poor man's dual signoff.